kotlin-native/litmus/src/commonMain/kotlin/AffinityBindings.kt

```diff
@@ -0,0 +1,17 @@
+import kotlin.native.concurrent.Worker
+
+interface AffinityManager {
+    fun setAffinity(w: Worker, cpus: Set<Int>)
+    fun getAffinity(w: Worker): Set<Int>
+
+    fun scheduleShort2(): List<AffinityMap>
+    fun scheduleLong2(): List<AffinityMap>
```

Comment:
What do these 2 functions do? Why does `Short` and `Long` here means? What does `2` mean?